doc/usage.rst

```diff
@@ -0,0 +1,10 @@
+Usage
+=====
+How to use the `expertsystem` to create models can be found in 
+<https://pwa.readthedocs.io/projects/tensorwaves/en/latest/usage/QuickStart.html>
```

Comment:
Beter use an 'internal' link here:
```rst
:doc:`usage/QuickStart`
```
Have a look at the cross-referencing syntax [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-syntax).